stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/event/handler/internal/EventHandlerImpl.kt

```diff
@@ -134,7 +134,7 @@ internal class EventHandlerImpl(
             val initJob = scope.async {
                 syncManager.updateAllReadStateForDate(currentUser.id, Date())
                 syncManager.loadSyncStateForUser(currentUser.id)
-                replayEventsForAllChannels(currentUser)
+                replayEventsForAllChannels()
```

Comment:
Hey @MarinTolic 

Do you mind reverting it back? It may cause merge conflict with changes, which touch Syncing & Event handling

Reply:
Reverted :smile: